util/lev_comp.l

```diff
@@ -230,6 +230,7 @@ paralyzed	ST_RET(PARALYZED_ID);
 stunned		ST_RET(STUNNED_ID);
 confused	ST_RET(CONFUSED_ID);
 seen_traps	ST_RET(SEENTRAPS_ID);
+dead		ST_RET(DEAD_ID);
```

Comment:
lev_comp.l no longer does anything in splicehack since it now uses lua scripting to load levels.

Reply:
i guess it'd make sense to remove that in another PR first, then? thanks!